pages/labTwo.html

```diff
@@ -1,39 +1,170 @@
-<!DOCTYPE html>
-<html lang="en">
+<!DOCTYPE HTML>
+<html lang="en-US">
 <head>
     <meta charset="UTF-8">
-    <title>Lab Project 2</title>
-    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Droid+Serif|Lobster" />
-    <style>
+    <title>Patient Information Site</title>
+    <style type="text/css">
         body {
-            background-color: #ffffcc;
+            margin: 2em 5em;
+            font-family: Georgia, "Times New Roman", Times, serif;
         }
 
-        body>p {
-            color: #006600;
-            margin-left: 20px;
-            font-size: 65px;
-            text-align: center;
-            font-family: "Lobster", cursive, serif;
+        h1, legend {
+            font-family: Arial, Helvetica, sans-serif;
         }
-        .footer {
-            bottom : 2px;
-            height : 40px;
-            margin-top : 40px;
-            text-align: center;
-            vertical-align: middle;
-            position:fixed;
-            width:100%;
+
+        label, input, select {
+            display: block;
+        }
+
+        input, select {
+            margin-bottom: 1em;
+        }
+
+        fieldset {
+            margin-bottom: 2em;
+            padding: 1em;
+        }
+
+        fieldset fieldset {
+            margin-top: 1em;
+            margin-bottom: 1em;
         }
     </style>
 </head>
 <body>
-<p> Coming soon!! Keep in touch</p>
-<div class="footer">
+<form id="register">
+    <h1>Patient Registration Form</h1>
+    <fieldset>
+        <legend>Personal Details</legend>
+        <div title="Use this field for identifying the patient by name or his/her patient ID">
+            <label for="given-name">Patient ID or Name</label>
+            <input id="given-name" name="given-name" type="text" placeholder="Patient ID or name only" required
+                   autofocus>
+        </div>
+
+        <div title="This field should represent the date/time when the patient data collection was started">
+            <label for="datenTime">Current date and time</label>
+            <input id="datenTime" name="datenTime" type="text" placeholder="UTC" required>
+        </div>
+
+        <table>
+            <label for="ptemp">Patient Temperature </label>
+            <tr>
+                <td><input id="ptemp" type="number" placeholder="100" required></td>
+                <td><input id="celcius" name="deg" type="radio"></td>
+                <td><label for="celcius"> &#x2103;</label></td>
+                <td><input id="fahrenheit" name="deg" type="radio"></td>
+                <td><label for="fahrenheit">&#x2109;</label></td>
+            <tr>
+        </table>
+        <div>
+            <table>
+                <label>Pulse Rate </label>
```

Comment:
Labels don't go here in tables... and you should probably do this (you are trying to space out the field and its text?) as inline or inline-block to avoid issues with different screen widths. Bootstrap can help here as well.